config.js

```diff
@@ -218,5 +218,6 @@ conf.set('mail.passwordResetUrl', contentServerUrl + '/v1/complete_reset_passwor
 conf.set('mail.initiatePasswordResetUrl', contentServerUrl + '/reset_password')
 conf.set('mail.initiatePasswordChangeUrl', contentServerUrl + '/settings/change_password')
 conf.set('mail.verifyLoginUrl', contentServerUrl + '/complete_signin')
+conf.set('mail.reportSignInUrl', contentServerUrl + '/reject_unblock_code')
```

Comment:
I wasn't sure whether to call this `reportSignInUrl` to be generic and allow for future expansion, or just simply `rejectUnblockCode`.